src/main.c

```diff
@@ -1,125 +1,130 @@
 #include "chip.h"
 #include "can.h"
 #include "ccand_11xx.h"
-#include "board.h"
+
+#include "adc.h"
+#include "serial.h"
+#include "can_constants.h"
 
 /*****************************************************************************
  * Private types/enumerations/variables
  ****************************************************************************/
 
-#define BAUDRATE 115200
-
 const uint32_t OscRateIn = 12000000;
 
+#define SERIAL_BAUDRATE 115200
+#define CAN_BAUDRATE 500000
+
 volatile uint32_t msTicks;
 
-uint32_t last_can_message_sent_time = 0; 
+const uint32_t adcMessagePeriod_ms = 1000 / FRONT_CAN_NODE_ANALOG_SENSORS__freq;
+const uint32_t rpmMessagePeriod_ms = 1000 / FRONT_CAN_NODE_WHEEL_SPEED__freq;
+
+uint32_t lastAdcMessage_ms = 0;
+uint32_t lastRpmMessage_ms = 0;
 
 /*****************************************************************************
  * Private function
  ****************************************************************************/
 
 void SysTick_Handler(void) {
-    msTicks++;
+  msTicks++;
 }
 
 /**
  * @details reads incoming CAN messages and prints information to the terminal
  */
 void Process_CAN_Inputs(void) {	
-	CCAN_MSG_OBJ_T rx_msg;
-	uint32_t ret = CAN_Receive(&rx_msg);
-
-        if (ret == NO_CAN_ERROR) {
-		Board_Println("Received CAN message.");
-
-		Board_Print("ID: ");
-		Board_Println_Int(rx_msg.mode_id, 10);
-
-		Board_Println("Data: ");
-		Board_Print("data[0]: ");
-		Board_Println_Int(rx_msg.data[0], 2);
-		Board_Print("data[1]: ");
-		Board_Println_Int(rx_msg.data[1], 2);
-		Board_Print("data[2]: ");
-		Board_Println_Int(rx_msg.data[2], 2);
-		Board_Print("data[3]: ");
-		Board_Println_Int(rx_msg.data[3], 2);
-		Board_Print("data[4]: ");
-		Board_Println_Int(rx_msg.data[4], 2);
-		Board_Print("data[5]: ");
-		Board_Println_Int(rx_msg.data[5], 2);
-		Board_Print("data[6]: ");
-		Board_Println_Int(rx_msg.data[6], 2);
-		Board_Print("data[7]: ");
-		Board_Println_Int(rx_msg.data[7], 2);
-
-	}
+  CCAN_MSG_OBJ_T rx_msg;
+  uint32_t ret = CAN_Receive(&rx_msg);
+
+  if (ret == NO_RX_CAN_MESSAGE) {
```

Comment:
Meant this whole section, changing it to not freak out if a return value other than the two defined in CAN_ERROR_T is observed.